core/src/avm2/globals/flash/display3D/Context3DBufferUsage.as

```diff
@@ -6,6 +6,7 @@
 package flash.display3D
 {
 
+    [API("692")]
```

Comment:
I think this one should be `694`, for Flash Player 12: https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display3D/Context3DBufferUsage.html